lab-isaac/src/reducer/expense.js

```diff
@@ -0,0 +1,27 @@
+let initialState = {};
```

Comment:
there are a lot of linter errors in here, I know you probably don't get the unexpected lexical declaration in code block, but a few of them are legitimate syntax linter errors. 

https://eslint.org/docs/2.0.0/rules/no-case-declarations